CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
```

Comment:
☝️ should change this to 7.2.0-wip (and in pubspec) since new features are being added

Reply:
done